### ISSUE TITLE
Dashboard unresponsive [Bug] 

### ISSUE THREAD

**Issue by ltnhawk** [IceWhaleTech/CasaOS#1646]
**Describe the bug**
The dashboard doesn't open apps or the app store

> A clear and concise description of what the bug is.

**To Reproduce**

> Steps to reproduce the behavior:

**Expected behavior**

> A clear and concise description of what you expected to happen.

**Screenshots**

> If applicable, add screenshots to help explain your problem.

**Desktop (please complete the following information):**

```
 - OS: [e.g. iOS]
 - Browser [e.g. chrome, safari]
 - Version [e.g. 22]
```

**System Time**

> Run `timedatectl` and share the output

```
(timedatectl output here)
```

**Logs**

> Run following command to collect corresponding logs:

```bash
sudo journalctl -xef -u casaos-gateway
sudo journalctl -xef -u casaos-user-service
sudo journalctl -xef -u casaos-local-storage
sudo journalctl -xef -u casaos-app-management
sudo journalctl -xef -u casaos.service
```

**Additional context**

> Add any other context about the problem here.
> 
> If you are a Zimaboard user, make it explicit with when you got your Zimaboard.

**Comment by CorrectRoadH** [comment]
Are All App Icon be gray? I think it may caused by mergeFS🤔

**Comment by ltnhawk** [comment]
No, they are not greyed out. I did do the disk merge. I have tried deleting and reinstalling casaos but the problem remains.

Jan 25, 2024 3:51:53 AM CorrectRoadH ***@***.***>:
